tsconfig.json

```diff
@@ -34,7 +35,8 @@
     "./www/assets/**/*.js",
   ],
   "exclude": [
-    "./base-theme/assets/webpack"
+    "./base-theme/assets/webpack",
+    "./dist/**/*"
```

Comment:
Since we only want to exclude js files, should we not add .js file. extension here?

Not sure, perhaps, you were also doing something else here. Otherwise looks good to me!

Reply:
I think you're right. Since in the dist directory, `.js.map` files are also generated, and all of our errors are only coming from `.js` files, there's no reason to exclude other files, such as `.js.map`